python/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+include(AddMLIRPython)
+
+# Specifies that all MLIR packages are co-located under the `gc_mlir`
+# top level package (the API has been embedded in a relocatable way).
+# TODO: Add an upstream cmake param for this vs having a global here.
+add_compile_definitions("MLIR_PYTHON_PACKAGE_PREFIX=gc_mlir.")
+
+
```

Comment:
can we add a cmake option to turn off the pybind even if LLVM has python enabled?

Reply:
> can we add a cmake option to turn off the pybind even if LLVM has python enabled?

yes， I can add a new  cmake option for the gc pybind, should we set the default value to OFF?

Reply:
@ZhennanQin Do we need pybind by default? Or should we only target onednn graph api for now?

Reply:
Suggest to enable pybind by default as this is the way for release. For develop purposes, you can turn it off with extra cmake flag.

Reply:
option added in https://github.com/intel/graph-compiler/blob/dda942cb46a0fffb658a3b360ab31fff970e7cd0/CMakeLists.txt#L54C1-L57C8